GATK_CNV_Mutect2/GATK4_CNV/GATK4_CNV.wdl

```diff
@@ -655,12 +655,20 @@ task DenoiseReadCounts {
         set -e
         export GATK_LOCAL_JAR=${default="/root/gatk.jar" gatk4_jar_override}
 
+        if [[ -f "${read_count_pon}" ]]; then
         gatk --java-options "-Xmx${command_mem_mb}m" DenoiseReadCounts \
             --input ${read_counts} \
             --count-panel-of-normals ${read_count_pon} \
```

Comment:
You can change this to `${'--count-panel-of-normals '+ read_count_pon}` and get rid of the if statement to make this a bit cleaner.